package.json

```diff
@@ -87,6 +87,9 @@
     "files": [
       "packages/*/test/**/test-*.js"
     ],
+    "nodeArguments": [
+      "--expose-gc"
+    ],
```

Comment:
Do you really want to enable this globally for all tests in the entire agoric-sdk or just for SwingSet?  I can see there being an argument for the former; I just want to be sure it's deliberate.

Reply:
Unfortunately I don't have a choice: now that the top-level `yarn test` runs AVA directly (which enables more parallelism), it no longer consults the individual `packages/*/package.json` files: every test program in the entire monorepo gets the same set of flags.

Also, it's not a problem to expose `gc` to the start compartment of all programs.. it doesn't enable that code to anything worse than it could already do.